vmngclient/api/templates.py

```diff
@@ -216,6 +223,33 @@ def create(self, device_model: DeviceModel, name: str, description: str, config:
             cli_template.config = config
             return cli_template.send_to_device()
 
+    def __validation_template(self, id: str, device: Device) -> bool:
```

Comment:
I would name it `__template_validation` as it is the action.
When it is `__validation_template` it sounds more like a template called `validation`. (like feature template or CLI template).